### PR TITLE
[Fix] get_file_list function fix

### PR DESCRIPTION
**Pull request by victor-paltz**
Fixing #11

### CODE REVIEW

**Review by hitchhicker — APPROVED** [review]
LGTM

File: embedding_reader/get_file_list.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def _get_file_list(
     path = make_path_absolute(path)
     fs, path_in_fs = fsspec.core.url_to_fs(path)
     prefix = path[: path.index(path_in_fs)]
-    glob_pattern = path.rstrip("/") + f"**/*.{file_format}"
```
Comment (rom1504) [inline embedding_reader/get_file_list.py:45]: this was definitely wrong

File: embedding_reader/get_file_list.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def _get_file_list(
...
+    glob_pattern = path.rstrip("/") + f"/**.{file_format}"
```
Comment (rom1504) [inline embedding_reader/get_file_list.py:45]: but this is also wrong

File: embedding_reader/get_file_list.py (hunk 1 above)
Comment (rom1504) [reply embedding_reader/get_file_list.py:45]: path = "my_folder"
path.rstrip("/") + f"**/*.{file_format}"
-> 'my_folder**/*.parquet'

File: embedding_reader/get_file_list.py (hunk 2 above)
Comment (rom1504) [reply embedding_reader/get_file_list.py:45]:
```
path = "my_folder"
path.rstrip("/") + f"/**.{file_format}"
->'my_folder/**.parquet'
```

Comment (rom1504) [reply embedding_reader/get_file_list.py:45]: we want path.rstrip("/") + f"/**/*.{file_format}"

Comment (rom1504) [reply embedding_reader/get_file_list.py:45]: ok actually I tried it and 'my_folder/**.parquet' does what we want

good then

Comment (victor-paltz) [reply embedding_reader/get_file_list.py:45]: In JavaScript, the Glob.js function would match "/x/y" with "/x/**/y"
But in python, both `glob.glob` and `fs.glob`  wouldn't match "/x/y", ** is the same as .* for a regex ([doc](https://docs.python.org/3/library/glob.html))
As you can see [here](https://github.com/fsspec/filesystem_spec/blob/3e739db7e53f5b408319dcc9d11e92bc1f938902/fsspec/spec.py#L542), they replace `**` with `.*`

**Pull request merged**